test/esinstall/export-maps/package.json

```diff
@@ -7,14 +7,17 @@
   },
   "snowpack": {
     "install": [
-      "preact"
+      "preact",
+      "preact/hooks",
+      "@clevercloud/components/assets/backup.svg"
```

Comment:
This adds a local test for the new behavior (the others are just snapshot updates)